docs/go-installation.md

```diff
@@ -14,14 +14,36 @@ go get -u go.undefinedlabs.com/scopeagent
 ## Instrumenting your tests and benchmarks
 
 In order to instrument your tests that use Go's native [`testing`](https://golang.org/pkg/testing/) package, you
-have to call `scopeagent.Run(m)` in your `TestMain` function, and all tests and benchmarks in the package will be instrumented.
+have to import `go.undefinedlabs.com/scopeagent/autoinstrument` and all tests and benchmarks in the package will be instrumented.
 
 For example:
 
 ```go
 import (
-    "go.undefinedlabs.com/scopeagent"
     "testing"
+
+    _ "go.undefinedlabs.com/scopeagent/autoinstrument"
+)
+
+func TestExample(t *testing.T) {
+    // ...
+}
+
+func BenchmarkExample(b *testing.B) {
+    // ...
+}
+```
+
+The automatic test instrumentation by imports is achieved using monkey patching, this can be disabled by adding the environment variable `SCOPE_DISABLE_MONKEY_PATCHING=true`. In case monkey patching is disabled, as an alternative, you can call `scopeagent.Run(m)` in your `TestMain` function.
```

Comment:
As I commented earlier, this is confusing. You should be able to disable autoinstrumentation by just not importing `autoinstrument`.

Reply:
Agree, that's simpler for the user... I'm gonna do a `PR` to do that refactor.

Reply:
done.